src/main/java/flab/delideli/controller/OrderController.java

```diff
@@ -22,13 +22,21 @@
 @RequestMapping("/orders")
 public class OrderController {
 
-	private OrderService orderService;
+	private OrderUserService orderService;
+	private CommonPaymentService commonPaymentService;
 
-	@PostMapping(value = "/{userId}")
+	@PostMapping()
 	@ResponseStatus(HttpStatus.CREATED)
 	@ApiOperation(value = "주문 등록")
 	public void registerOrder(@RequestBody @Valid RequestOrderDTO requestOrderDTO, @CurrentUser String userId) {
 		orderService.registerOrder(requestOrderDTO, userId);
 	}
 
+	@PostMapping(value = "/{orderId}/cancel")
+	@ApiOperation(value = "주문 취소")
+	public void cancelOrder(@CurrentUser String userId, @PathVariable  long orderId) {
```

Comment:
check style이 동작하지 않은 것 같네요~

```suggestion
	public void cancelOrder(@CurrentUser String userId, @PathVariable long orderId) {
```

Reply:
Code With Me로 작업하다 보니 체크가 안 된 부분이 있는 것 같습니다 확인하겠습니다~

Reply:
넵, 확인해서 수정하겠습니다~